statsmodels/graphics/functional.py

```diff
@@ -638,24 +644,44 @@ def fboxplot(data, xdata=None, labels=None, depth=None, method='MBD',
     # Plot envelope of all non-outlying data
     lower_nonout = data[ix_nonout, :].min(axis=0)
     upper_nonout = data[ix_nonout, :].max(axis=0)
+    ax.plot(xdata, lower_nonout,
```

Comment:
Should the have the same color as the fill? `(0.75, 0.75, 0.75)`?

Reply:
Thanks a lot for your comments, they can be the same color, but the different colors would be easier for users to observe the marginal line. If `'blue'` is not that good, would you mind to set as `(0, 0, 1)`?

Reply:
It is more a question of whether the style should change across versions without any warning of future changes.  I don't mind that much since users can always modify the plot after.